data/abilities/collection/90c2efaa-8205-480d-8bb6-61d90dbaf81b.yml

```diff
@@ -11,28 +11,28 @@
     darwin:
       sh:
         command: |
-          find /Users -name '*.#{file.sensitive.extension}' -type f -not -path '*/\.*' -size -500k 2>/dev/null | head -5
+          find /Users -name '*.#{file.is_sensitive}' -type f -not -path '*/\.*' -size -500k 2>/dev/null | head -5
         parsers:
           plugins.stockpile.app.parsers.basic:
-            - source: host.file.path
+            - source: file.path
               edge: has_extension
-              target: file.sensitive.extension
+              target: file.is_sensitive
```

Comment:
this name almost makes it seem like a boolean, but we're really looking for extensions, right?

Reply:
I can see how this can be seen as a bool. I believe this should captures the action better:

![image](https://user-images.githubusercontent.com/81784737/163654748-537086db-ae9a-4e01-8539-1d6536e9a6c3.png)

Reply:
could we keep the `has_extension` edge and use a tertiary tag like `file.extension.sensitive`?